src/virtual-machine.js

```diff
@@ -336,6 +336,13 @@ class VirtualMachine extends EventEmitter {
         });
     }
 
+    /*
+     * @type {Array<object>} Array of all costumes and sounds currently in the runtime
+     */
+    get assets () {
+        return collectAssets(this.runtime);
+    }
```

Comment:
Since `collectAssets` reaches into the runtime to do its work and this looks like the only place it's called from, it seems like it would make sense to merge the guts of `collectAssets` into this accessor.

Reply:
Good point. It used to do more — you could filter by a target — but then I decided to only make it do what we need.